docs/interpreter/bigquery.md

```diff
@@ -48,6 +48,11 @@ limitations under the License.
     <td>100000</td>
     <td>Max result set size</td>
   </tr>
+  <tr>
+    <td>zeppelin.bigquery.sql_dialect</td>
+    <td></td>
+    <td>Bigquery SQL dialect (standardSQL or legacySQL). If empty, query prefix like '#standardSQL' can be used.</td>
```

Comment:
is there a link we could add to the details of these dialects?

Reply:
yes, please add them into this PR.

also should we quote the values, ie. "standardSQL" and if they are case sensitive